examples/md-flexible/scripts/generate-mdflex-configs.py

```diff
@@ -8,42 +33,46 @@
 except ImportError:
     import itertools
 
+
 # Script to generate a wide range of yaml input files intended to be used as benchmarks
 # to judge the adaptability of tuning strategies.
-
 template = """
-container                        :  [LinkedCells, LinkedCellsReferences, VarVerletListsAsBuild, VerletClusterLists, VerletLists, VerletListsCells, PairwiseVerletLists Octree]
+functor                          :  Lennard-Jones (12-6) AVX
+# container                        :  [all]    # Currently unfeasible. See below.
+# Containers that are excluded:
+#   - DirectSum             : Takes too long
+#   - LinkedCellsReferences : Quasi equivalent to LinkedCells
+#   - Octree                : Takes too long
+#   - PairwiseVerletLists   : Uses too much memory resulting in chrashes
+container                        :  [LinkedCells, VerletLists, VarVerletLists, VerletListsCells, VerletClusterLists]
 verlet-rebuild-frequency         :  20
-verlet-skin-radius               :  0.15
+verlet-skin-radius-per-timestep  :  0.0075
```

Comment:
I think this is now out of date for the master

Reply:
-> 630a150